### PR TITLE
Ag grid sample

### PR DESCRIPTION
**Pull request by Jennnyyy**

### CODE REVIEW

File: src/pages/home/HomePage.tsx [hunk 1]
```diff
@@ -5,6 +5,7 @@ export const HomePage: React.FunctionComponent = props => {
   return (
     <Container style={{ textAlign: "center" }}>
       <div>
+        {/* <AgGridReact/> */}
```
Comment (ao508) [inline src/pages/home/HomePage.tsx:8]: delete

File: src/pages/requests/RequestSummary.tsx [hunk 2]
```diff
@@ -53,6 +64,80 @@ const RequestSummary: FunctionComponent<IRequestSummaryProps> = ({
   const [typingTimeout, setTypingTimeout] = useState<any>(null);
   const [prom, setProm] = useState<any>(Promise.resolve());
 
+  const [rowData, setRowData] = useState([
+    { cmoSampleName: "", cmoPatientId: "", investigatorSampleId: "" },
+    { cmoSampleName: "", cmoPatientId: "", investigatorSampleId: "" }
+  ]);
```
Comment (ao508) [inline src/pages/requests/RequestSummary.tsx:70]: are these meant to be here? I am a little unclear with what these are for.

File: src/pages/requests/RequestSummary.tsx [hunk 3]
```diff
@@ -53,6 +64,80 @@ const RequestSummary: FunctionComponent<IRequestSummaryProps> = ({
...
+      sortable: true,
+      filterable: true
+    },
+    {
+      headerName: "Tissue Location",
+      field: "tissueLocation",
+      sortable: true,
+      filterable: true
+    },
+    { headerName: "Sex", field: "sex", sortable: true, filterable: true }
```
Comment (ao508) [inline src/pages/requests/RequestSummary.tsx:138]: looks like these are the sample headers? Can we move these to the `helpers.tsx` or do they need to be explicitly defined here now due to the move to AgGrid?

File: src/pages/requests/RequestSummary.tsx [hunk 4]
```diff
@@ -70,38 +155,15 @@ const RequestSummary: FunctionComponent<IRequestSummaryProps> = ({
...
-    </AutoSizer>
-  );
+    return sm || {};
+
+    // if (
+    //   request.hasSampleSamples?.[index]?.hasMetadataSampleMetadata?.[0] !== undefined
+    // ) {
+    //   return request.hasSampleSamples[index].hasMetadataSampleMetadata[0];
+    // }
+  }
```
Comment (ao508) [inline src/pages/requests/RequestSummary.tsx:166]: some more cleanup

File: src/pages/requests/RequestSummary.tsx [hunk 5]
```diff
@@ -116,13 +178,15 @@ const RequestSummary: FunctionComponent<IRequestSummaryProps> = ({
...
+  const remoteCount = request.hasSampleSamples.length;
+
   return (
     <>
       {showDownloadModal && (
         <DownloadModal
           loader={() => {
             return Promise.resolve(
-              CSVFormulate(metadataList, SampleDetailsColumns)
+              CSVFormulate(metadataList, oldSampleDetailsColumns)
```
Comment (ao508) [inline src/pages/requests/RequestSummary.tsx:189]: this is probably something else that we can clean up

File: src/pages/requests/RequestSummary.tsx [hunk 6]
```diff
@@ -182,7 +250,10 @@ const RequestSummary: FunctionComponent<IRequestSummaryProps> = ({
           </Button>
         </Col>
       </Row>
-      <div style={{ height: 540 }}>{sampleTable}</div>
+      {/* <div style={{ height: 540 }}>{sampleTable}</div> */}
```
Comment (ao508) [inline src/pages/requests/RequestSummary.tsx:253]: this line can be removed also I believe

File: src/pages/requests/requests.scss [hunk 7]
```diff
@@ -21,12 +21,13 @@
...
+
+// .ReactVirtualized__Table__row {
+//   border-bottom: 1px solid #e0e0e0;
+// }
+// .ReactVirtualized__Table__row:nth-child(odd) {
+//   background-color: #ffffff;
+// }
+// .ReactVirtualized__Table__row:nth-child(even) {
+//   background-color: #fafafa;
+// }
```
Comment (ao508) [inline src/pages/requests/requests.scss:33]: remove

**Review by ao508 — CHANGES_REQUESTED** [review]
Generally speaking I would say that there is quite a bit to clean up from these code changes. 

Some general guidelines to follow when making a pull request:
1. Avoid commenting out code that isn't going to be used anymore and just delete it instead. [example](https://github.com/mskcc/smile-dashboard/pull/39/files#diff-6cf796d3f4c2efe3675aec3d6f37245c41c005617b11c6830fd687e8175338baL24-R33)

2. Clean up and/or remove any code that was added just to "test things out". [example](https://github.com/mskcc/smile-dashboard/pull/39/files#diff-cddd5a8bd2c026c7a59decefef75c33d3cd070c848f3320d2564dca44266833aR58-R59)

3. Removing comments throughout the code that appear to be left behind for one reason or another. [example](https://github.com/mskcc/smile-dashboard/pull/39/files#diff-a2ca9dc69117526ee7e40339495edb1fc5a0f42e690aadc1a90c377fc7e8ef45R189)

4. Choose variable names with their intention made clear. [example](https://github.com/mskcc/smile-dashboard/pull/39/files#diff-a2ca9dc69117526ee7e40339495edb1fc5a0f42e690aadc1a90c377fc7e8ef45R298) this I think was left behind when ironing out issues with getting the data to render properly in the sample table

File: src/pages/requests/RequestSummary.tsx (hunk 2 above)
Comment (Jennnyyy) [reply src/pages/requests/RequestSummary.tsx:70]: I removed.

**Pull request merged**